src/controllers/teams.controller/deleteTeamDocument/index.js

```diff
@@ -0,0 +1,61 @@
+const { validationResult } = require('express-validator');
+const catchAsync = require('../../../utils/catchAsync');
+const { query: Hasura } = require('../../../utils/hasura');
+const { getTeamDocumentQuery, checkIfMemberQuery, deleteTeamDocumentQuery } = require('./queries/queries');
+const deleteDocument = require('../../../utils/deleteDocument');
+
+const deleteTeamDocument = catchAsync(async (req, res) => {
+  const sanitizerErrors = validationResult(req);
+  if (!sanitizerErrors.isEmpty()) {
+    return res.status(400).json({
+      success: false,
+      errors: sanitizerErrors.array(),
+    });
+  }
+
+  const { cognito_sub: cognitoSub, team_id: teamId, documentId } = req.body;
```

Comment:
Since this is a new API & apoorv doesn't have to change anything on his end, lets directly use teamId